pages.de/windows/wsl.md

```diff
@@ -0,0 +1,36 @@
+# wsl
+
+> Verwalte das Windows Subsystem für Linux von der Kommandozeile.
+> Weitere Informationen: <https://docs.microsoft.com/windows/wsl/reference>.
+
+- Starte eine Linux-Shell (in der Standard-Distribution):
+
+`wsl {{shell_kommando}}`
+
+- Führe ein Linux-Kommando aus ohne eine Shell zu benutzen:
+
+`wsl --exec {{kommando}} {{kommando_argumente}}`
+
+- Gib eine bestimmte Distribution an:
+
+`wsl --distribution {{distribution}} {{shell_kommando}}`
+
+- Liste verfügbare Distributionen auf:
+
+`wsl --list`
+
+- Exportiere eine Distribution in eine `.tar` Datei:
+
+`wsl --export {{distribution}} {{pfad/zu/tar_datei.tar}}`
```

Comment:
I'm not sure what `distro_fs` means in the English page, but the `tar_` part here is reduntant anyways. How about
```suggestion
`wsl --export {{distribution}} {{pfad/zu/distro.tar}}`
```
Or just
```suggestion
`wsl --export {{distribution}} {{pfad/zu/datei.tar}}`
```